docs/dashboard/configuration/attributes.md

```diff
@@ -2,5 +2,95 @@
 id: attributes
 title: Attributes
 ---
+## About Attributes
+
+The Attributes page allows you to use the full capacity of the product structure in Saleor. By configuring your products right, product management in Saleor becomes an hassle-free task.
+
+The main Attributes page displays a list of all existing attributes. The list consist of several columns indicating the attributes' parameters (such as an attribute code or whether the attribute is visible to your customers).
+
+You can create variations of products you sell to reflect the wide range of colours, sizes or material the product was made from.
+
+For more information on the product structure in Saleor, see the ![Products](dashboard/catalog/products.md#product-concept-introduction) topic.
+
+> **Example**
+>
+> To illustrate what is the role of an Attribute, consider the following example:
+> 
+> Product Category: Shoes
+> 
+> Product Type: Women Sport Shoes
+> 
+> Product: NB Women Running Shoes
+> 
+> At this stage you sell pink and black NB Women Running Shoes.
+> 
+> Both these colours come in sizes ranging from 36 to 40.
+> 
+> We have then two attributes at hand: 
+> 
+> * shoe size
+> * colour
+> 
+> The available **Attribute Values** are in this case:
+> 
+> * For Attribute _shoe size_: 36, 37, 38, 39 and 40.
+> * For Attribute _colour_: pink and black.
+> 
+> The **Variant Attributes** are a combination of an Attribute and its specific Value.
+>
+> Variant Attributes are created according to the following pattern:
+> 
+> Variant Attribute = Attribute_1 (shoe size: 36) + Attribute Value_1 (colour: pink)
+>
+> This way you can create an entire matrix of colours and sizes and make them available to your customers.
+
+
+### How to add an attribute
+
+1. In the main Attribute page, click [[Addnbsp;Attribute]]. 
+2. In the General Information card, provide the following information:
+
+*  Default Label - This is the name of your attribute visible to your customers. For example, size. The default label must be a unique value.
```

Comment:
Attribute codes must be unique, while labels can be duplicated. E.g.:
- label "Size", code "shoe-size"
- label "Size", code "cushion-size"